generate-config/cmd.go

```diff
@@ -17,7 +17,7 @@ func newCommand() *cobra.Command {
 	opt := configGenOpts{}
 
 	cmd := &cobra.Command{
-		Use:   "config-gen",
+		Use:   "generate-config",
```

Comment:
Could we use `cmd/generate-config/`? It more consistent with go dir pattern and doesn't pollute root dir

Reply:
Done